src/Service/TripalCultivateImportValidationHelper.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+namespace Drupal\trpcultivate\Service;
+
+/**
+ * Import validation helper class.
+ */
+class TripalCultivateImportValidationHelper {
+
+  /**
+   * A mapping of supported file mime-types and their supported delimiters.
+   *
+   * More specifically, the file is split based on the appropriate delimiter
+   * for the mime-type passed in. For example, the mime-type
+   * "text/tab-separated-values" maps to the tab (i.e. "\t") delimiter.
+   *
+   * By using this mapping approach, we can actually support a number of
+   * file types with different delimiters for the same importer while keeping
+   * the performance hit to a minimum. Especially since in many cases this is a
+   * one-to-one mapping. If it is not a one-to-one mapping, then we loop through
+   * the options.
+   *
+   * @var array
```

Comment:
We should be more specific when documenting an array and now seems like a good time to improve this one :-) What about:

```suggestion
   * Official mime types are managed by IANA.
   * @see https://www.iana.org/assignments/media-types/media-types.xhtml
   *
   * @var array
   *   An associative array where the key is the official mime type and the
   *   value is an array of supported delimiters for that mime type. Delimiters
   *   should be enclosed in double quotes for consistency.
```

Reply:
We don't need to look elsewhere for mapping 👍

Reply:
Applied in https://github.com/TripalCultivate/TripalCultivate/pull/64/commits/571027e7f2b90ebeda13abbd4f9be30df75ba659. The suggestion was manually applied instead of using the Commit suggestion button. This is because of the service file name change, and the suggestion wanted to write to a file that no longer exists.